src/itest/kotlin/nl/info/zac/itest/ZaakKoppelenRestServiceTest.kt

```diff
@@ -44,12 +46,26 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
                 zaakUUID = getString("uuid").let(UUID::fromString)
             }
         }
+        itestHttpClient.performGetRequest(
+            "$ZAC_API_URI/zaken/zaak/id/ZAAK-2000-0000000003"
```

Comment:
Maybe make `ZAAK-2000-0000000003` into a global constant too? Seems a bit magical to use it here. I assume that this zaak was created in some previous itest but without a constant it is hard to judge.

Reply:
Extracted as constant.

We don't really use this for creating the zaak, so creation place still remains a mistery. Third call with date in 2020.